frontend/src/components/index.ts

```diff
@@ -0,0 +1 @@
+export {Body} from './body/Body'
```

Comment:
src/components mein no need of index file

Reply:
okay